CMakeLists.txt

```diff
@@ -47,6 +47,8 @@ lxqt_translate_ts(QM_FILES
     ${UIS}
     INSTALL_DIR
     "${LXQT_TRANSLATIONS_DIR}/${PROJECT_NAME}"
+    PULL_TRANSLATIONS ${PULL_TRANSLATIONS}
```

Comment:
@palinek There is cmake -LA 
I'm not against it. I placed it there to save the boring work of putting it in every top CMakeLists